### ISSUE TITLE
Create a mentorship pool

### ISSUE THREAD

**Issue by jeff-zucker** [solid/deit#12]
There was a lot of talk in the first DEI Team meeting about how people feel intimidated by the tech.  Maybe we should have a pool of people who would be willing to mentor beginners in a welcoming way.  I volunteer to try.

**Comment by ewingson** [comment]
yep, I volunteer, too. once we have a beginners-room I would bookmark it and give my best (although I' m on the way myself still and it's hard to explain what you haven't caught...)

**Comment by KyraAssaad** [comment]
We [discussed this](https://hackmd.io/eiS6Mqu4ToWHDYwP0iHA3A?view) at the DEI Team Meeting today (Aug 10). We thought that gauging interest in a buddy system might be a good first step, as it is less intimidating than a formal mentorship program. 

Action items: 
- [ ] Post a poll in the forum to determine interest in a buddy program
